spec/factories/courses.rb

```diff
@@ -62,5 +68,39 @@
     trait :resulting_in_pgde do
       qualification { :pgde }
     end
+
+    trait :with_study_mode_as_full_time do
+      study_mode { :full_time }
+    end
+
+    trait :with_study_mode_as_part_time do
+      study_mode { :part_time }
+    end
+
+    trait :with_study_mode_as_full_time_or_part_time do
+      study_mode { :full_time_or_part_time }
+    end
+
+    factory :course_with_site_statuses do
+      with_study_mode_as_full_time_or_part_time
+
+      transient do
+        site_statuses_traits {
+          [
+            [:findable],
```

Comment:
How come these need to be arrays? Can we not make them just symbols and not splat it when we call the factory?